src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -77,11 +77,11 @@
 					<a
 						:href="`https://${this.$appConfig.host}/legal/privacy`"
 						target="_blank"
-						title="Open Privacy Policy in a new window"
-					>Privacy Policy</a>.
+						:title="`Open Privacy ${enableCommsExperiment ? 'Notice' : 'Policy' } in a new window`"
+					>Privacy {{ enableCommsExperiment ? 'Notice' : 'Policy' }}</a>.
 					<p v-if="$v.termsAgreement.$error" class="input-error tw-text-danger tw-text-base">
 						You must agree to the Kiva Terms of service & Privacy
-						policy.
+						{{ enableCommsExperiment ? 'Notice' : 'Policy' }}.
```

Comment:
It looks like "policy" was lowercase before, is having it always uppercase expected?